Error: Cannot find module 'truffle-hdwallet-provider'

`Error: Cannot find module 'truffle-hdwallet-provider'`

### Reason:
`truffle-hdwallet-provider` has been deprecated and replaced with `@truffle/hdwallet-provider`

### Solution
Use `npm i @truffle/hdwallet-provider`
and `npm i fs` in the directory containing **truffle-config.js**

Also don't forget to make the following changes in **truffle-config.js**:
`const HDWalletProvider = require('truffle-hdwallet-provider');`
changes to
`const HDWalletProvider = require('@truffle/hdwallet-provider');`

### Fix
I'll make a pull request and fix this.

--- a/docs/develop/truffle.md
+++ b/docs/develop/truffle.md
@@ -29,7 +29,7 @@ To verify that Truffle is installed properly, type **`truffle version`** on a 
 - Update the truffle-config with matic-network-crendentials.
 
 ```js
-const HDWalletProvider = require('truffle-hdwallet-provider');
+const HDWalletProvider = require('@truffle/hdwallet-provider');
 const fs = require('fs');
 const mnemonic = fs.readFileSync(".secret").toString().trim();
 

